server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/PartitionPlanController.java

```diff
@@ -28,38 +27,36 @@
 import com.oceanbase.odc.service.common.response.Responses;
 import com.oceanbase.odc.service.common.response.SuccessResponse;
 import com.oceanbase.odc.service.partitionplan.PartitionPlanService;
-import com.oceanbase.odc.service.partitionplan.model.ConnectionPartitionPlan;
+import com.oceanbase.odc.service.partitionplan.model.DatabasePartitionPlan;
 
 /**
  * @Author：tianke
  * @Date: 2022/9/16 15:38
  * @Descripition:
  */
 @RestController
-@ConditionalOnProperty(name = "odc.feature.partitionplan.enabled", havingValue = "true")
 public class PartitionPlanController {
 
     @Autowired
     private PartitionPlanService partitionPlanService;
 
-    @RequestMapping(value = "/api/v2/patitionplan/ConnectionPartitionPlan", method = RequestMethod.GET)
-    public SuccessResponse<ConnectionPartitionPlan> getConnectionPartitionPlan(@RequestParam Long connectionId,
+    @RequestMapping(value = "/api/v2/patitionplan/DatabasePartitionPlan/", method = RequestMethod.GET)
```

Comment:
seems no RESTful, what is the relationship between database and partition plan ?

may `api/v2/partitionplan/PartitionPlans/{id}` ?

Reply:
The API has been modified to 'api/v2/partitionplan/partitionPlans'